cmd/uhc/cluster/describe/describe.go

```diff
@@ -29,23 +30,45 @@ import (
 
 var args struct {
 	debug bool
+	out   string
+	short bool
 }
 
 var Cmd = &cobra.Command{
-	Use:   "describe CLUSTERID",
+	Use:   "describe CLUSTERID [--out] [--short]",
 	Short: "Decribe a cluster",
 	Long:  "Get info about a cluster identified by its cluster ID",
 	Run:   run,
 }
 
+// Check error
+func check(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
+
 func init() {
+	// Add flags to rootCmd
 	flags := Cmd.Flags()
 	flags.BoolVar(
 		&args.debug,
 		"debug",
 		false,
 		"Enable debug mode.",
 	)
+	flags.StringVar(
+		&args.out,
+		"out",
+		"",
+		"Output result into json file.",
+	)
+	flags.BoolVar(
+		&args.short,
+		"short",
+		false,
+		"Output a simpler description of the defined cluster",
```

Comment:
This is named _short_, but it is actually longer :-) . I think it should rather be `-o json` or `--output json`.

Reply:
@jhernand 
I am confused...  -- short outputs the old default: 
```
ID:       1IMuyxEDNaAdjObqb9m61cobCCn
Name:     3224f265-eeb2-486e-9d17-b9a8eeebcf8c.
Masters:  0
Computes: 0
Region:   
Multi-az: false
```
without this flag we would get the entire body of a cluster